src/stories/RedPackets.tsx

```diff
@@ -110,7 +111,7 @@ storiesOf('Plugin: Red Packets', module)
                         mnemonic: ['mem'],
                         name: 'Wallet Name',
                         passphrase: 'password',
-                        eth_balance: BigInt(2 * 10 ** 18),
+                        eth_balance: new BigNumber(2).multipliedBy(10).pow(18), // (2 * 10) ** 18
```

Comment:
Wait... `2 * 10 ** 18` is `2 * (10 ** 18)`

Reply:
Thanks